packages/core/src/test/codewhispererChat/tools/fsRead.test.ts

```diff
@@ -0,0 +1,118 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import assert from 'assert'
+import * as vscode from 'vscode'
+import { FsRead } from '../../../codewhispererChat/tools/fsRead'
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const TEST_DIR_NAME = 'fsread-tests'
+
+describe('FsRead Unit Tests', () => {
+    let rootUri: vscode.Uri
+    let testDirUri: vscode.Uri
+
+    before(async () => {
+        const workspaceFolders = vscode.workspace.workspaceFolders
+        if (!workspaceFolders || workspaceFolders.length === 0) {
+            throw new Error('No workspace folder is open. Please open a folder in VS Code before running tests.')
+        }
+        rootUri = workspaceFolders[0].uri
+
+        testDirUri = vscode.Uri.joinPath(rootUri, TEST_DIR_NAME)
```

Comment:
If you're looking to create test folders you can just use: https://github.com/aws/aws-toolkit-vscode/blob/master/packages/core/src/test/testUtil.ts#L77

Reply:
Thank you for pointing me to this!